### ISSUE TITLE
Activation of plugin slows down telescope

### ISSUE THREAD

**Issue by StephenGunn** [dmmulroy/tsc.nvim#53]
Hi, just want to say that this plugin is a life-saver for inheriting a big typescript project that has errors in it. I am newer to neovim and am coming from vscode. I am only opening this issue after trying many things on my own without any change.. but it still might be something stupid I am overlooking.

I am using the Lazy plugin manager. 

When I activate this plugin, the opening of telescope is slowed down by 500-1000ms, which feels very sluggish. 

I have tried it with nvim-notify and without, both result in the same telescope experience.

If it matters, I am using Alacritty / Tmux / Manjaro Linux.

Let me know if I can assist in any way, thank you!

**Comment by dmmulroy** [comment]
🤔 Interesting, I don't seemingly encounter any lag w/ telescope - it's possible its your PCs hardware. The type checking done via `tsc` (the actual TS compiler not the plugin) runs in a completely separate thread from Neovim and subsequently telescope. Do you encounter this on even small/fresh/new TS repos?

Could you also share your neovim config?

**Comment by StephenGunn** [comment]
Yeah, I messed with it for a while and didn't see any change. As soon as I disabled the plugin, telescope went back to being instant.

It didn't matter what repo I was in, it was even slow in my /dotfiles. I am running a desktop computer that is powerful, but a few years old. I am just assuming it's some edge-case that you don't need to spend your time trying to debug.

Here is my dotfiles repo, always a work in progress: https://github.com/StephenGunn/dotfiles

On another note, it did push me to try my hand at neovim plugin development.. I threw together a plugin that runs `svelte-check` and populates the results in the quickfix menu: https://github.com/StephenGunn/sveltecheck.nvim - gave you inspiration credit on the readme :)

Appreciate any time you spent on the issue but it's not a big deal, I can enable the plugin on a case by case basis if I need to.

Thanks
